flow/project.py

```diff
@@ -2951,7 +2955,13 @@ def _add_print_status_args(cls, parser):
         parser.add_argument(
             '--no-parallelize',
             action='store_true',
-            help="Do not parallelize the status determination.")
+            help="Do not parallelize the status determination. "
+                 "This feature is deprecated, you should set "
+                 "the flow.status_parallelization config value "
+                 "to 'none', 'thread', 'process'. You can do this by "
+                 "executing `signac config set flow.status_parallelization "
+                 "['none'|'thread'|'process']`."
+            )
```

Comment:
Please document the feature as part of the general documentation for the `status` command and then only refer to that here. Otherwise this information will disappear after we completely remove this option.

Reply:
Then I'll update this CLI document after documenting the general documentation.